README.spanish.md

```diff
@@ -757,15 +806,15 @@ Todo lo de encima hubiera devuelto `false` si se hubiera usado `===`
 
 **De lo contrario:** La entrada de usuario no validada o no higiénica podría conducir a la inyección del operador cuando se trabaja con MongoDB para NoSQL, y no usar un sistema de desinfección adecuado u ORM permitirá fácilmente ataques de inyección SQL, creando una vulnerabilidad gigante.
 
-🔗 [**Leer más: Query injection prevention using ORM/ODM libraries**](/sections/security/ormodmusage.spanish.md)
+🔗 [**Leer más: Prevención de inyección de consultas utilizando bibliotecas ORM/ODM**](./sections/security/ormodmusage.spanish.md)
 
 <br/><br/>
 
 ## ![✔] 6.5. Colección de mejores prácticas de seguridad genéricas
 
-**TL;DR:** Esta es una colección de consejos de seguridad que no está relacionada directamente con Node.js: la implementación de Node no es muy diferente a la de cualquier otro idioma. Haz clic en leer más para hojear.
+**TL;DR:** Esta es una colección de consejos de seguridad que no está relacionada directamente con Node.js: la implementación de Node no es muy diferente a la de cualquier otro idioma. Haz clic en leer más para mas información.
```

Comment:
Hojear is more a regional term. And besides that most sections in here (and in general) uses the "para más información"